files/scripts/arp_update

```diff
@@ -93,6 +93,18 @@ while /bin/true; do
         ping6_template="sed -e 's/^/timeout 0.2 ping /' -e 's/,/ -n -q -i 0 -c 1 -W 1 -I /' -e 's/$/ >\/dev\/null;/'"
         failed_ip6_neigh_cmd="ip -6 neigh show | grep -v fe80 | grep $vlan | grep -E 'FAILED|INCOMPLETE' | cut -d ' ' -f 1,3 --output-delimiter=',' | $ping6_template"
         eval `eval $failed_ip6_neigh_cmd`
+
+        # manually set any remaining FAILED/INCOMPLETE entries to permanently INCOMPLETE
+        # this prevents any remaining INCOMPLETE entries from automatically transitioning to FAILED
+        # once these entries are incomplete, any subsequent neighbor advertisement messages
+        # are able to resolve the entry
+
+        # generates the following command for each failed or incomplete IPv6 neighbor
+        # ip neigh replace <neighbor IPv6> dev <VLAN name> nud incomplete
+        neigh_replace_template="sed -e 's/^/ip neigh replace /' -e 's/,/ dev /' -e 's/$/ nud incomplete;/'" 
+        ip_neigh_replace_cmd="ip -6 neigh show | grep -v fe80 | grep Vlan1000 | grep -E 'FAILED|INCOMPLETE' | cut -d ' ' -f 1,3 --output-delimiter=',' | $neigh_replace_template"
+        eval `eval $ip_neigh_replace_cmd`
```

Comment:
Let's do this prior to sending ping (line 95). We don't want to end up in a situation where ping succeeded, then set to INCOMPLETE by the script due to some timing issue. So lets first set to INCOMPLETE and then send ping which is safe.